test/benchmark/avro-upsert/create_views/views/insert_views.sql

```diff
@@ -0,0 +1,29 @@
+-- Copyright Materialize, Inc. All rights reserved.
+--
+-- Use of this software is governed by the Business Source License
+-- included in the LICENSE file at the root of this repository.
+--
+-- As of the Change Date specified in that file, in accordance with
+-- the Business Source License, use of this software will be governed
+-- by the Apache License, Version 2.0.
+
+CREATE SOURCE source_upsertavrotest
```

Comment:
maybe we should have a different name for these just because they aren't upsert anymore? maybe ingest_filtered or something like that?

Reply:
Thansk -- I changed this to be `source_insertavrotest`.